src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -230,6 +238,56 @@ private function updateGetOperation(\ArrayObject $pathOperation, array $mimeType
     }
 
     /**
+     * @param string $resourceClass
+     * @param string $operationName
+     *
+     * @return array
+     */
+    protected function getPaginationParameters(string $resourceClass, string $operationName)
```

Comment:
If it adds nothing more than existing type hints and return types, it is useless and should be removed to ease maintenance.